src/inject/dynamic-theme/inline-style.ts

```diff
@@ -168,7 +170,14 @@ function deepWatchForInlineStyles(
     });
     treeObservers.set(root, treeObserver);
 
-    const attrObserver = new MutationObserver((mutations) => {
+    let attributecount = 0;
```

Comment:
Rename this variable to have a proper meaning. Is it attempts count> Use camel case.